### ISSUE TITLE
[Accessibility Bug] Self Identification Search input Screen Reader Innacurate

### ISSUE THREAD

**Issue by AminaSalah** [CDH-Studio/UpSkill#1551]
**Describe the bug**
For the Self Identification Search input – if you type in an ‘a’ there will be some options that appear. 
1. “Woman” 
2. “Indig…”
3. “Person with disability”
4. “Member of…”

The screen reader reads the numbering of the options (e.g., Option 1 of 2) but it doesn’t always match up with what options appear. If you type 'a' and use the arrow go down the list you will notice that the screen reader reads the numbering inaccurately. E.g., it will read the:
1. first option “Woman” as option 1/2, 
2. the second “Indig…”2/3, 
3. third “Person with disability”2/3, 
4. the fourth “Member of…”2/2.

**Expected behavior**
The numbering should be as follows if the user types in the letter 'a'
1. first option “Woman” as option 1/4, 
2. the second “Indig…”2/2, 
3. third “Person with disability”3/4, 
4. the fourth “Member of…”4/4.

Note: A similar issue exists if you type 'b'

**Comment by dynamic11** [comment]
I am working on this currently. We are switching our drop down to react-select:
https://react-select.com/home
